packages/client/src/containers/RetroPage/RetroPage.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { apiURL } from "../../apiURL";
+import PropTypes from 'prop-types'; 
+import './RetroPage.css';
+
+
+export default function RetroPage({ retroCode, setRetroCode }) {
+  const navigate = useNavigate();
+  const [currentDate, setCurrentDate] = useState("");
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  const updateTeamInformation = () => {
+    navigate("/updateTeamPage");
+  }
+
+  const initializeRetroSession = () => {
+    if (!retroCode || retroCode.length !== 10) {
+      setError('Retro code must be 8 characters');
+      return;
+    }
+    const newRetroCode = Math.random().toString(36).substring(2, 10);
+    setRetroCode(newRetroCode);
+    setTimeout(() => {
+    navigate(`/RetroPage2/${newRetroCode}`);
```

Comment:
What is `RetroPage2` ?

Reply:
another component/page that direct the user or a route to that page if that makes sense